[2024-12-30 11:41:20,352] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag_V1.second_task manual__2024-12-30T11:41:18.551571+00:00 [queued]>
[2024-12-30 11:41:20,359] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag_V1.second_task manual__2024-12-30T11:41:18.551571+00:00 [queued]>
[2024-12-30 11:41:20,359] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-12-30 11:41:20,359] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2024-12-30 11:41:20,359] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-12-30 11:41:20,369] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): second_task> on 2024-12-30 11:41:18.551571+00:00
[2024-12-30 11:41:20,373] {standard_task_runner.py:52} INFO - Started process 11376 to run task
[2024-12-30 11:41:20,376] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag_V1', 'second_task', 'manual__2024-12-30T11:41:18.551571+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp9xo5jvt5', '--error-file', '/tmp/tmprnnz_su6']
[2024-12-30 11:41:20,376] {standard_task_runner.py:80} INFO - Job 12: Subtask second_task
[2024-12-30 11:41:20,415] {task_command.py:370} INFO - Running <TaskInstance: first_dag_V1.second_task manual__2024-12-30T11:41:18.551571+00:00 [running]> on host 0766066e7c2f
[2024-12-30 11:41:20,476] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=swetansu
AIRFLOW_CTX_DAG_ID=first_dag_V1
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2024-12-30T11:41:18.551571+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-12-30T11:41:18.551571+00:00
[2024-12-30 11:41:20,477] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-12-30 11:41:20,477] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo i am task 2 and will be running after task 1']
[2024-12-30 11:41:20,485] {subprocess.py:85} INFO - Output:
[2024-12-30 11:41:20,486] {subprocess.py:92} INFO - i am task 2 and will be running after task 1
[2024-12-30 11:41:20,487] {subprocess.py:96} INFO - Command exited with return code 0
[2024-12-30 11:41:20,510] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag_V1, task_id=second_task, execution_date=20241230T114118, start_date=20241230T114120, end_date=20241230T114120
[2024-12-30 11:41:20,547] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-12-30 11:41:20,571] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
